llvm/lib/ProfileData/InstrProf.cpp

```diff
@@ -352,6 +366,8 @@ std::string getIRPGOFuncName(const Function &F, bool InLTO) {
   return getIRPGOObjectName(F, InLTO, getPGOFuncNameMetadata(F));
 }
 
+// DEPRECATED. Use `getIRPGOFuncName`for new code. See that function for
```

Comment:
In the header it is described as for FE instrumentation. Probably want the comments to be consistent?

Reply:
done.